pg_search/src/postgres/customscan/pdbscan/exec_methods/fast_fields/mod.rs

```diff
@@ -220,13 +224,29 @@ fn collect_fast_field_try_for_attno(
             if let Some(att) = tupdesc.get((attno - 1) as usize) {
                 if let Some(search_field) = schema.search_field(att.name()) {
                     if search_field.is_fast() {
-                        let ff_type = if att.type_oid().value() == pg_sys::TEXTOID
-                            || att.type_oid().value() == pg_sys::VARCHAROID
-                            || att.type_oid().value() == pg_sys::UUIDOID
-                        {
-                            FastFieldType::String
-                        } else {
-                            FastFieldType::Numeric
+                        let ff_type = match att.type_oid().value() {
```

Comment:
How's our test coverage around this `match` statement?

Nothing that should hold up this PR, which looks just fine and good, but if we don't have tests that exercise all the types we claim to support as fast fields we should probably put one together.

Reply:
I think that the most valuable test coverage that we could probably have for this match would be https://github.com/paradedb/paradedb/issues/2903 ... which I've marked as a "good first issue", because it's basically pure SQL and `proptest` crate shenanigans.

It's likely that `arrow_array_to_datum` should have a dedicated unit/prop-test though: I'll look into that before landing this.

Reply:
Have added some property tests that attempt to ensure alignment with `TantivyValue`, and moved `arrow_array_to_datum` nearby.